pages/learn/develop/hardware/gpio.md

```diff
@@ -11,23 +11,34 @@ Recommended ways of interacting with GPIO on {{ $names.company.lower }} devices.
 
 ## Raspberry Pi
 
-The [Raspberry Pi][rpi]'s [General Purpose I/O][gpio] (GPIO) pins can be used to send and receive arbitrary data from external hardware. In the diagram shown below the GPIO pins are located in the top-right hand corner of the device:-
+All Raspberry Pi SBCs have a header that includes a number of [General Purpose Input/Output ("GPIO") pins](https://en.wikipedia.org/wiki/General-purpose_input/output). These pins can be used to control external hardware such as LEDs, solenoids, motors, or other devices. (The GPIO pins employ standard 3.3 volt logic and can't drive high current devices directly.) The earliest models used a 26 pin header, but all recent models use a 40 pin header. 
```

Comment:
As well as "controlling external hardware", the GPIO pins can also be used to "read data from external sensors" ? :wink: